package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "poa-parity-bridge-contracts",
+  "name": "token-bridge-contracts",
   "version": "1.0.0",
```

Comment:
Similar to the comment above.

Reply:
I'll update the version on `package` and `package-lock` files to the current version. There are some tools to automate all the publishing workflow but they seems overkill for our needs. I think we could use [npm version](https://docs.npmjs.com/cli/version) to generate the new releases. Here are some possible steps:

1. Run `npm version <newversion>`. This command will:
- Update `version` on `package.json` and `package-lock.json`
- Commit the changes
- Create a new git tag for the new version

2. Push the commit and tag `git push && git push --tags` (we can skip this manual step if we define a `postversion` script to do that)
3. Create the new GitHub Release using the new tag

Reply:
Thanks for clarification. Probably we will use this tool in the future.

Reply:
Updated version https://github.com/poanetwork/poa-bridge-contracts/pull/230/commits/617f62004ca4c404c5ca27f1c423fe625d26a4a4